internal/pkg/cli/project_delete.go

```diff
@@ -129,6 +129,10 @@ func (o *deleteProjOpts) Execute() error {
 		return err
 	}
 
+	if err := o.deleteProjectPipeline(); err != nil {
+		return err
+	}
+
```

Comment:
Noting for reviewers: This call must happen before `o. deleteProjectResources` and `o. deleteLocalWorkspace`, since the pipeline delete command relies on the project stackset as well as the workspace directory to still exist.

Reply:
:+1: can you move this feedback to a comment in the code so that a future dev knows not to mess with the order?
Another cool move would be to have a test to ensure this property is maintained but I see that our `project_delete_tests` don't really exist 😭